examples/attachments.rs

```diff
@@ -25,10 +25,12 @@ async fn main() {
     .to(BOB_DID.to_owned())
     .from(ALICE_DID.to_owned())
     .attachment(Attachment {
-        data: AttachmentData::Json(JsonAttachmentData {
-            json: json!({"foo": "bar"}),
-            jws: None,
-        }),
+        data: AttachmentData::Json {
```

Comment:
As i understand we decided to use Json serialisation. Do we need such changes?

Reply:
No, we don't use JSON serialization. We were able to generate all types via uniffi.